### DISCUSSION TITLE
Outbound ports for automatic updates

### DISCUSSION THREAD

**Discussion by begunfx** [ESPresense/ESPresense#176] [General]
I am setting up a bunch of the Atom Lite M5 stack esp32, and from what I understand they will get automatic updates (assuming that option was left enabled).  My question is: what port or ports do they need for outbound traffic for updates?  The reason I'm asking is I keep my network outbound traffic locked down, and only open ports for known devices.  

Btw, kudos on this project!  Awesome work.  I'm really excited to get these up and running.

**Comment by stevencoutts** [comment]
HTTPS to github.

Regards

Sent from my iPhone

On 12 Jan 2022, at 04:09, Brian Begun ***@***.***> wrote:

﻿

I am setting up a bunch of the Atom Lite M5 stack esp32, and from what I understand they will get automatic updates (assuming that option was left enabled). My question is: what port or ports do they need for outbound traffic for updates? The reason I'm asking is I keep my network outbound traffic locked down, and only open ports for known devices.

Btw, kudos on this project! Awesome work. I'm really excited to get these up and running.

—
Reply to this email directly, view it on GitHub<https://github.com/ESPresense/ESPresense/discussions/176>, or unsubscribe<https://github.com/notifications/unsubscribe-auth/ALYN2WFCEIKGMXY2GE2F4UDUVT5FZANCNFSM5LX6SLMA>.
Triage notifications on the go with GitHub Mobile for iOS<https://apps.apple.com/app/apple-store/id1477376905?ct=notification-email&mt=8&pt=524675> or Android<https://play.google.com/store/apps/details?id=com.github.android&referrer=utm_campaign%3Dnotification-email%26utm_medium%3Demail%26utm_source%3Dgithub>.

**Reply by begunfx** [reply]
Okay, so just port 443 then.  Cool.  Thanks!
